import-service/src/handlers/import-file-parser.js

```diff
@@ -0,0 +1,39 @@
+import csv from 'csv-parser';
+import { readStreamObject, deleteObject, copyObject, } from '../services/s3-client';
+
+const readObject = (params) => new Promise((res, rej) => {
+  readStreamObject(params)
+    .pipe(csv())
+    .on('data', chunk => {
+      console.info('Read row of data', JSON.stringify(chunk, null, 2));
+    })
+    .on('end', () => res('Stream close'))
+    .on('error', e => {
+      console.info('Error occurred during object reading');
+      rej(e);
+    });
+})
+
+export const importFileParser = async (event) => {
+  console.info('Event', JSON.stringify(event));
+
+  try {
+    const { bucket, object } = event.Records[0].s3;
```

Comment:
better to create iteration - because it's not ok idea to use 0 index only and only one record

Reply:
You will always receive only one item for each event because ObjectCreate triggers this event, what point to use iteration?

Reply:
I thought that it could be more than one, because there are array. I googled it and really they provide always one object in the array. Good catch and thanks)